src/utils/signupValidation.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+/* eslint-disable no-restricted-syntax */
+export default class Validator {
```

Comment:
The validation put in place here should actually come from the backend hence the use of the FAILED_SIGNUP DISPATCH ACTION because I feel this redundant in a way that we already put this validation in the backend

Reply:
@kodek-sleuth I think we must handle the validation of user information on the frontends. Because if a user is going to wait for validation of input from the backend It can take a long time to get it.